manifests/a/Atlassian/Sourcetree/3.4.22/Atlassian.Sourcetree.installer.yaml

```
@@ -0,0 +1,30 @@
+# Created with komac v2.10.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.9.0.schema.json
+
+PackageIdentifier: Atlassian.Sourcetree
+PackageVersion: 3.4.22
+InstallerLocale: en-US
+Scope: user
+InstallModes:
+- interactive
+- silent
+- silentWithProgress
+UpgradeBehavior: install
+ReleaseDate: 2025-02-04
+Installers:
+- Architecture: x64
+  InstallerType: wix
+  InstallerUrl: https://product-downloads.atlassian.com/software/sourcetree/windows/ga/SourcetreeEnterpriseSetup_3.4.22.msi
+  InstallerSha256: C20318332A081452694FD0FFD5B6DF7BA05A2A43B020B2878B09F2E290EE2712
+  InstallerSwitches:
+    Custom: ACCEPTEULA=1
+  ProductCode: '{8A756017-B20E-444E-A184-2DFF6CEA85FF}'
+  AppsAndFeaturesEntries:
+  - UpgradeCode: pnd0\expndtw1\hich\af8\dbch\af8\afs20\

```

```suggestion
  - UpgradeCode: {E18DCCFE-AFC4-42A5-A020-1FF11BF12D39}
```

This will probably be the same bug as https://github.com/mdsteele/rust-msi/pull/18